docs/config.md

```diff
@@ -267,3 +267,18 @@ Disables the p2p-circuit relay transport.
 - `EnableRelayHop`
 Enables HOP relay for the node. If this is enabled, the node will act as
 an intermediate (Hop Relay) node in relay circuits for connected peers.
+
+### `ConnMgr`
+Connection manager configuration.
+
+- `Type`
+Sets the type of connection manager to use, options are: `"none"` and `"basic"`.
+
+- `LowWater`
+LowWater is the minimum number of connections to maintain.
+
+- `HighWater`
+HighWater is the number of connections that, when exceeded, will trigger a connection GC operation.
```

Comment:
I don't really like these names. Wouldn't simple `Low`/`High` be enough?

Reply:
Hrm... I'd something a little more descriptive than that. I've seen these terms used in this context before. But i'm open to suggestions

Reply:
How about a simple `Min` and `Max` or maybe `MinConn` or `MaxConn`?

Reply:
`Watermark` is something that is familiar to me in terms of setting upper and lower bounds to something in programming. No special feelings attached though.

Reply:
Note: I'd avoid min/low as they imply that we'll actively seek peers up to this point.

Reply:
So... are we in favor of still using `HighWater` and `LowWater` ? I'm getting mixed signals (especially from @Stebalien who thumbs upped davids comment, but suggested changing things). 

I agree that "min" would imply we actively open connections until we have that many, but I think "low" avoids this connotation.

Reply:
`(GC)Treshold(Conn/Peers)`, `(GC)Target(Conn/Peers)`? `Water` may sound confusing for some even if it's technically correct.